pages/votepage/[id].js

```diff
@@ -0,0 +1,62 @@
+import { useRouter } from "next/router";
+import styled from "styled-components";
+import { useState } from "react";
+import { StyledWrapper } from "@/components/styles/Wrapper";
+import { StyledBackLink } from "@/components/styles/BackLink";
+import VoteDetails from "@/components/VoteDetails";
+function Votepage({ entryData, onHandleUpdateVoteEvent }) {
+  const [emojiCheckmark, setEmojiCheckmark] = useState("");
+  const router = useRouter();
+  const { id } = router.query;
+
+  const currentEvent = entryData.find((entryToVote) => entryToVote.id === id);
+
+  function handleVoteResult() {
+    const resultVoteEvent = { ...currentEvent, voteResult: emojiCheckmark };
+
+    onHandleUpdateVoteEvent(resultVoteEvent);
+  }
+
+  return (
+    <StyledWrapper>
+      <h2>{currentEvent?.title}</h2>
+      {emojiCheckmark}
+      <VoteDetails
+        emojiCheckmark={emojiCheckmark}
+        currentEvent={currentEvent}
+        entryData={entryData}
+      ></VoteDetails>
```

Comment:
can be self closing:

```suggestion
      <VoteDetails
        emojiCheckmark={emojiCheckmark}
        currentEvent={currentEvent}
        entryData={entryData}
      />
```